src/d3a/d3a_core/sim_results/stats.py

```diff
@@ -107,6 +110,26 @@ def _get_child_data(self, area):
                         for child in area.children}
             return self.bills_results[area.name]
 
+    def _store_area_penalties(self, results_dict, area):
```

Comment:
Correct me if Im wrong but this method would be called at every_tick?

Reply:
No, this is called on market cycle